Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1081,4 +1091,56 @@ namespace AtomSampleViewer
         }
 
     }
+
+    void ScriptReporter::ExportTestResults()
+    {  
+        for (const ScriptReport& scriptReport : m_scriptReports)
+        {
+
+            const AZStd::string& assertLogLine = AZStd::string::format("Asserts: %u \n", scriptReport.m_assertCount);
```

Comment:
Instead of using
const AZStd::string&
it should be
const AZStd::string

This error should be corrected for all the strings in this functiuon